jetty-util/src/main/java/org/eclipse/jetty/util/Pool.java

```diff
@@ -548,34 +574,17 @@ public String toString()
      */
     public class Entry
     {
-        // hi: positive=open/maxUsage counter; negative=closed; MIN_VALUE pending
-        // lo: multiplexing counter
+        // hi: negative==closed else total-usage counter; negative=closed; MIN_VALUE pending
+        // lo: in-use counter
         private final AtomicBiInteger state;
```

Comment:
If we got rid of the two counters this `AtomicBiInteger` keeps track of, we could simplify it down to a tri-state `PENDING / OPEN / CLOSED` that would measurably boost performance of highly contended pools.